### PR TITLE
Fix the snapshot log info

### PR DESCRIPTION
**Pull request by panpan0000**
Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?

Fixes #(issue) NA

The original log will be quite confusing for PV(c) bakcup users.

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: pkg/backup/item_backupper.go [hunk 1]
```diff
@@ -545,7 +545,8 @@ func (ib *itemBackupper) takePVSnapshot(obj runtime.Unstructured, log logrus.Fie
 	}
 
 	if volumeSnapshotter == nil {
-		log.Info("Persistent volume is not a supported volume type for snapshots, skipping.")
+		// the PV may still has change to be snapshotted by CSI plugin's `PVCBackupItemAction` in PVC backup logic
+		log.Info("Persistent volume is not a supported volume type for built-in volumeSnapshotter snapshots, skipping.")
```
Comment (ywk253100) [inline pkg/backup/item_backupper.go:549]: Let's call it the `Velero-native volumeSnapshotter` instead

Comment (panpan0000) [reply pkg/backup/item_backupper.go]: Thx. fixed

**Comment by panpan0000** [comment]
@ywk253100 , when you have time, please review again. thx

**Review by ywk253100 — APPROVED** [review]

**Pull request merged**